txnbuild/transaction.go

```diff
@@ -269,6 +269,20 @@ func (t *Transaction) HashHex(network string) (string, error) {
 	return hashHex(t.envelope, network)
 }
 
+// Generates TranscationParam from a Transcation, Pass your own increment sequence flag.
+func (t *Transaction) ConvertToTransactionParam(incrementSeqFlag bool) *TransactionParams {
```

Comment:
I would rename it and don't return a pointer. Pointers are hairier to manage and, when returning them, aren't necessarily faster than copying since they usually cause an allocation.

```suggestion
func (t *Transaction) GeTransactionParams(incrementSeqFlag bool) TransactionParams {
```